google_gke_infra/README.md

```diff
@@ -148,97 +152,99 @@ module "k8s" {
 
-This value configures the version of the Control Plane. It needs to be upgraded first. Generally this operation takes ~15 minutes. Once that is done, re-run `terraform plan` to validate that no further changes are needed by Terraform. If the node pools are configured for auto upgrade, GCP will take care of automatically upgrading the nodes within the upcoming weeks to match the master version, and hence you don't need to manually adjust `node_version`.
+The updating operation takes about 15 minutes. Once it is done, re-run `terraform plan` to validate that Terraform does not need any further changes. If the node pools are configured for auto upgrade, GCP automatically upgrades the Nodes within the upcoming weeks to match the master version, so that you don't need to manually adjust the **node_version**.
 
-#### Update `node_version`
+#### Update **node_version**
 
-Once the Control Plane has been updated, set this value to the Master version as you see it in GCP. You need to explicitly state the **full semantic version string** for the nodes (ie `1.11.5-gke.4`). If you don't provide the patch version, (ie, you only use `1.11`), this will cause a permadiff in Terraform.
+Once the Control Plane has been updated, set this value to the master version as you see it in GCP. You need to explicitly state the **full semantic version string** for the Nodes, such as `1.11.5-gke.4`. Providing only the main version, such as `1.11`, will cause a permadiff in Terraform.
 
-Finally, run `terraform apply`, and GCP will update the nodes one at a time. This operation can take some time depending on the size of your nodepool. If you find that Terraform times out, simply wait for the operation to complete in GCP and re-run `terraform plan` and `terraform apply` to reconcile the state. You can supply a higher timeout value if needed (the default is 30 minutes - see [timeouts](#timeouts)).
+Finally, run `terraform apply` for GCP to update the nodes one at a time. This operation can take some time depending on the size of your node pool. If Terraform times out, wait for the operation to complete in GCP and rerun `terraform plan` and `terraform apply` to reconcile the state. If necessart, you can provide a [timeout](#timeouts) value higher than the default 30 minutes.
 
 ## Variables
 
-For more info, please see the [variables file](variables.tf).
+For details on variables, see the [this](variables.tf) file.
 
-### Required Variables
+### Required variables
 
 | Variable  | Description                                  |
 | :-------- | :------------------------------------------- |
-| `name`    | Name to use as a prefix to all the resources. |
-| `region`  | The region that hosts the cluster. Each node will be put in a different availability zone in the region for HA. |
+| **name**   | The name to use as a prefix for all the resources. |
+| **region**  | The region that hosts the cluster. Each Node will be put in a different availability zone in the region for HA. |
```

Comment:
`HA` meaning? I think we should use the full name :)

Reply:
here it means High Availability